src/algebra/ordered_monoid.lean

```diff
@@ -785,33 +785,57 @@ class linear_ordered_cancel_comm_monoid (α : Type u)
   extends ordered_cancel_comm_monoid α, linear_ordered_comm_monoid α
 
 section covariant_class_mul_le
-variables [cancel_comm_monoid α] [linear_order α] [covariant_class α α (*) (≤)]
+variables [linear_order α]
+
+section has_mul
+
+variable [has_mul α]
+
+section left
+
+variable [covariant_class α α (*) (≤)]
```

Comment:
Maybe to match the style already present here:
```suggestion
section has_mul
variable [has_mul α]

section left
variable [covariant_class α α (*) (≤)]
```
which makes it easier to match variables to sections